### DISCUSSION TITLE
Statistik - Bagaimana interpretasi boxplot berikut?

### DISCUSSION THREAD

**Discussion by Hertzspr** [teamalgoritma/community#32] [Q&A] [answered]
Apakah bisa dibilang terlalu banyak outlier di boxplot berikut dengan titik-titik yang banyak di atas batas?
Apakah cukup menggunakan median sebagai estimate nilai tengah jika outliernya sebanyak itu?
![Fare Boxplot](https://user-images.githubusercontent.com/95233522/160573922-ce20f17b-b94a-4632-a8d9-e5f0adb260b9.png)

Hasil dari dput():
list(stats = structure(c(0, 7.8958, 14.4542, 31.275, 65), .Dim = c(5L, 
1L)), n = 1308, conf = structure(c(13.4328305557682, 15.4755694442318
), .Dim = 2:1), out = c(71.2833, 263, 146.5208, 82.1708, 76.7292, 
80, 83.475, 73.5, 263, 77.2875, 247.5208, 73.5, 77.2875, 79.2, 
66.6, 69.55, 69.55, 146.5208, 69.55, 113.275, 76.2917, 90, 83.475, 
90, 79.2, 86.5, 512.3292, 79.65, 153.4625, 135.6333, 77.9583, 
78.85, 91.0792, 151.55, 247.5208, 151.55, 110.8833, 108.9, 83.1583, 
262.375, 164.8667, 134.5, 69.55, 135.6333, 153.4625, 133.65, 
66.6, 134.5, 263, 75.25, 69.3, 135.6333, 82.1708, 211.5, 227.525, 
73.5, 120, 113.275, 90, 120, 263, 81.8583, 89.1042, 91.0792, 
90, 78.2667, 151.55, 86.5, 108.9, 93.5, 221.7792, 106.425, 71, 
106.425, 110.8833, 227.525, 79.65, 110.8833, 79.65, 79.2, 78.2667, 
153.4625, 77.9583, 69.3, 76.7292, 73.5, 113.275, 133.65, 73.5, 
512.3292, 76.7292, 211.3375, 110.8833, 227.525, 151.55, 227.525, 
211.3375, 512.3292, 78.85, 262.375, 71, 86.5, 120, 77.9583, 211.3375, 
79.2, 69.55, 120, 93.5, 80, 83.1583, 69.55, 89.1042, 164.8667, 
69.55, 83.1583, 82.2667, 262.375, 76.2917, 263, 262.375, 262.375, 
263, 211.5, 211.5, 221.7792, 78.85, 221.7792, 75.2417, 151.55, 
262.375, 83.1583, 221.7792, 83.1583, 83.1583, 247.5208, 69.55, 
134.5, 227.525, 73.5, 164.8667, 211.5, 71.2833, 75.25, 106.425, 
134.5, 136.7792, 75.2417, 136.7792, 82.2667, 81.8583, 151.55, 
93.5, 135.6333, 146.5208, 211.3375, 79.2, 69.55, 512.3292, 73.5, 
69.55, 69.55, 134.5, 81.8583, 262.375, 93.5, 79.2, 164.8667, 
211.5, 90, 108.9), group = c(1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 
1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 
1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 
1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 
1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 
1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 
1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 
1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 
1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1), names = "1")

**Comment by inytss** [comment]
Dear Bapak Husada, pada boxplot tersebut sudah dapat dikatakan bahwa outliernya cukup banyak. Oleh karena itu, untuk menginterpretasikan nilai pemusatan data dapat menggunakan nilai median/nilai tengah agar lebih menggambarkan pusat datanya.

**Reply by Hertzspr** [reply]
Terima kasih.

**Comment by NabiilahArdini** [comment, marked as answer]
Bila melihat data Anda, selain memiliki banyak outlier, distribusinya juga terlihat jelas bersifat skewed, sehingga pemilihan median sudah tepat. 

Bila Anda ingin menggunakan pendekatan lain, mungkin bisa menggunakan **mean dri data yang sudah dibuang nilai outlier nya**. Hal ini agar dapat menggunakan beberapa nilai pada data, tapi tidak mengikut sertakan outlier. Berikut contoh tahapannya. Bisa gunakan fungsi `boxplot()` dan subset data biasa:

```
# mendapatkan nilai outlier dari kolom var1
boxplot_data <- boxplot(data$var1)
nilai_outlier <- boxplot_data$out

# menghilangkan yang (`-which`) var1 nya termasuk nilai outlier
data_nooutlier <- data[-which(data$var1 %in% nilai_outlier),]

# cari mean
mean(data_nooutlier$var1)
```

**Reply by Hertzspr** [reply]
Terima kasih.
